packages/integration-sdk-core/src/data/createIntegrationIngestionConfig.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  StepExecutionContext,
+  IntegrationIngestionConfigFieldMap,
+  Step,
+  IntegrationIngestionConfigData,
+} from '../types';
+
+/**
+ * Generates an ingestionConfig with childIngestionSources taking into account
+ * the integration steps that come as argument.
+ * The childIngestionSources will be the list of stepIds that have any dependencies
+ * on the steps that match the ingestion sources specified.
+ *
+ * @export
+ * @template TStepExecutionContext
+ * @param {IntegrationIngestionConfigData} ingestionConfigData ingestionData without childIngestionSources
+ * @param {Step<TStepExecutionContext>[]} integrationSteps total list of integration steps
+ * @return {*}  {IntegrationIngestionConfigFieldMap} ingestionData with childIngestionSources
+ */
+export function createIntegrationIngestionConfig<
```

Comment:
This looks good. 
What are your thoughts on how this method is called? I'm hoping that the SDK will provide a single exported method that's referencing a wrapper method of this with its args already provided. Something like `getIntegrationIngestionConfig()`